topostats/tracing/disordered_tracing.py

```diff
@@ -254,6 +255,17 @@ def smooth_mask(
         LOGGER.debug(f"[{self.filename}] : smoothing done by dilation {dilation_iterations}")
         return self.re_add_holes(grain, dilation, holearea_min_max)
 
+    @staticmethod
+    def calculate_dna_width(smoothed_mask: npt.NDArray, pruned_skeleton: npt.NDArray, px2nm: float = 1) -> float:
+        """
+        Calculate the average width in metres of the DNA using the trace and mask.
```

Comment:
Needs a more complete docstring (one of the reasons pre-commit checks are failing).

I've switched to the more explicit `mean` to avoid ambiguity with `mode` or `median`. More generally though this could be an option to have either the mean or the median width returned. :thinking: 

```suggestion
        Calculate the mean width in metres of the DNA using the trace and mask.
        
        Parameters
        ----------
        smoothed_mask : npt.NDArray
            Smoothed mask to be measured.
        pruned_skeleton : npt.NDArray
            Pruned skeleton.
        px2nm : float
            Scaling of pixels to nanometres.
            
        Returns
        -------
        float
            Width of DNA in metres.
```